Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -148,10 +148,10 @@ typedef void(^VOKManagedObjectsReturnBlock)(VOKArrayOfManagedObjects *managedObj
                                forManagedObjectContext:(nullable NSManagedObjectContext *)contextOrNil;
 
 /*
- Returns one entite matching the predicate. Asserts the count is exactly 1. If more objects are returned this method will let you know.
+ Returns one entity matching the predicate. Asserts the count is exactly 1. If more objects are returned, the last one returned by Core Data will be returned.
```

Comment:
It is correct that the last one is returned but I don't know that it makes sense to remove the part that tells you that it hits the assert if so. I'm fine with it though, since it says "Asserts the count is exactly 1" before